net-wireless/linssid/linssid-2.9.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit qmake-utils xdg-utils
+
+DESCRIPTION="Graphical wireless scanning for Linux"
+HOMEPAGE="https://sourceforge.net/projects/linssid/"
+SRC_URI="mirror://sourceforge/${PN}/${PN}_${PV}.orig.tar.gz"
```

Comment:
Please make tarball name match unpacked name of ${S}

Reply:
Why? Is this some sort of a policy? Sorry to ask, as I am confused, since I am hearing this the first time.

Reply:
I looked at how tarball is packed, seems it's not really possible to fully match ${S}, but at least top level dir can match. Anyway, It's a suggestion, not a requirement.

No, it's not not a policy, just sane thing to do.
if you unpack `foo-1.2.3.tar.bz2` you expect to have `foo-1.2.3` directory as a result.
Just less surprise that way.

Reply:
Well okay, I see what you mean. But that's the sense of `${S}` for me, to set it, when needed, like in this case. But on the other hand some other devs told me not to modify the archive name, only in cases like versionname only archives.

Reply:
opinions =)
ok since it's has non-standard structure and and another dir inside let's leave it as is.